src/main/java/uk/gov/hmcts/et/common/model/ccd/types/RespondentSumType.java

```diff
@@ -81,10 +81,14 @@ public class RespondentSumType {
     private String responseRespondentEmail;
     @JsonProperty("responseRespondentContactPreference")
     private String responseRespondentContactPreference;
+
     @JsonProperty("responseReceived")
     private String responseReceived;
     @JsonProperty("responseReceivedDate")
     private String responseReceivedDate;
+    @JsonProperty("responseReceivedCount")
+    private Integer responseReceivedCount; // for WA Tasks
```

Comment:
Does Integer work okay? Back of my mind there might be a reason we always use String

Reply:
It has worked in my testing but their might be some use cases I am not aware of. I'll ask Harpreet.

Reply:
As advised by Harpreet, I will change to String.